openedx/core/djangoapps/config_model_utils/admin.py

```diff
@@ -65,13 +66,12 @@ def key_fields(self):
     def stackable_fields(self):
         return list(self.model.STACKABLE_FIELDS)
 
-    @property
-    def config_fields(self):
-        fields = super(StackedConfigModelAdmin, self).get_fields(request, obj)
+    def get_config_fields(self, request, obj=None):
+        fields = super().get_fields(request, obj)
         return [field for field in fields if field not in self.key_fields]
 
     def get_fields(self, request, obj=None):
-        return self.key_fields + self.config_fields
+        return self.key_fields + self.get_config_fields(request, obj)
```

Comment:
This was another "this would just crash" before -- request and obj were not defined in `config_fields`... Only place `config_fields` was used, so it should be safe to redefine like this.